corehq/apps/users/tests/retire.py

```diff
@@ -65,6 +63,9 @@ def tearDown(self):
         super(RetireUserTestCase, self).tearDown()
 
     def test_retire(self):
+        with override_settings(UNIT_TESTING=False):
+            with self.assertRaisesMessage(ValueError, "Missing deleted_by"):
+                self.commcare_user.retire(self.domain, deleted_by=None)
```

Comment:
ideally these should be separate test cases

Reply:
https://github.com/dimagi/commcare-hq/pull/29902/commits/4896a9139b3972002c7fc19719ce8ff958e0be8d